src/ontology/mondo-edit.obo

```diff
@@ -395960,18 +395957,17 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 
 [Term]
 id: MONDO:0021016
-name: channelopathy
+name: obsolete channelopathy
 def: "A disease caused by disturbed function of ion channel subunits or the proteins that regulate them." [Wikipedia:Channelopathy]
 comment: Reason: out of scope specific to autosomal disorder. Term to consider: none.
-subset: obsoletion_candidate
 synonym: "disorder of ion channel activity" EXACT [MONDO:patterns/basis_in_disruption_of_process]
 synonym: "ion channel activity disease" EXACT [MONDO:design_pattern]
-xref: MESH:D053447 {source="MONDO:equivalentTo"}
-xref: UMLS:C1720983 {source="MONDO:equivalentTo"}
-intersection_of: MONDO:0000001 ! disease or disorder
-intersection_of: disease_has_basis_in_disruption_of GO:0005216 ! ion channel activity
+xref: MESH:D053447 {source="MONDO:obsoleteEquivalent"}
+xref: UMLS:C1720983 {source="MONDO:obsoleteEquivalent"}
 relationship: seeAlso https://github.com/monarch-initiative/mondo/issues/4116
+property_value: IAO:0000231 "out of scope" xsd:string {source="MONDO:excludeGroupingClass"}
 property_value: IAO:0006012 "2022-06-01" xsd:string
```

Comment:
@sabrinatoro this should be removed when the term is obsoleted. I'll take care of it